docs/development/best-practices.md

```diff
@@ -32,6 +32,7 @@ Read the [GitHub Docs, renaming a branch](https://docs.github.com/en/repositorie
   - Only use `istanbul` comments for unreachable code coverage that is needed for `codecov` completion
   - Use descriptive `istanbul` comments
 - Avoid cast or prefer `x as T` instead of `<T>x` cast
```

Comment:
@viceice can you make a suggestion  to update this line?

Reply:
not sure, maybe leave as is for now